query/transformation.go

```diff
@@ -42,7 +42,7 @@ func transformTimeseries(series api.Timeseries, transform transform, parameters
 // timerangeScale takes an api.Timerange and computes the "scale" or duration of one sample (in seconds).
 // It is useful for transformations that normalize on time (like derivative or integral).
 func timerangeScale(timerange api.Timerange) float64 {
-	return float64(timerange.Resolution)
+	return float64(timerange.Resolution())
```

Comment:
The `timerangeScale` function will convert to seconds (rather than milliseconds) once I make those changes everywhere (time units are still inconsistent for transformations/expressions I think) so there will be a division also.